packages/cli/src/commands/service/list.ts

```diff
@@ -14,38 +13,38 @@ export default class ServiceList extends Command {
 
   async run(): Promise<IInstance[]> {
     const {flags} = this.parse(ServiceList)
-    const [{services}, {instances}, {runners}] = await Promise.all([
-      this.api.service.list({}),
-      this.api.instance.list({}),
-      this.api.runner.list({}),
+    const [services, instances, runners] = await Promise.all([
+      this.lcd.service.list(),
+      this.lcd.instance.list(),
+      this.lcd.runner.list(),
     ])
-    cli.table(services || [], {
-      hash: {header: 'HASH', get: srv => base58.encode(srv.hash)},
+    cli.table(services, {
+      hash: {header: 'HASH', get: srv => srv.hash},
       sid: {header: 'SID', get: srv => srv.sid},
       instances: {
         header: 'INSTANCES',
-        get: srv => (instances || [])
-          .filter(inst => inst.serviceHash.toString() === srv.hash.toString())
+        get: srv => instances
+          .filter(inst => inst.serviceHash === srv.hash)
           .map(inst => [
-            base58.encode(inst.hash),
-            (runners || [])
-              .filter(run => run.instanceHash.toString() === inst.hash.toString())
+            inst.hash,
+            runners
+              .filter(run => run.instanceHash === inst.hash)
               .reduce((p, _, i) => p + (i > 0 ? '\n' : ''), '')
           ].join(''))
         .join('\n'),
       },
       runners: {
         header: 'RUNNERS',
-        get: srv => (instances || [])
-          .filter(inst => inst.serviceHash.toString() === srv.hash.toString())
-          .map(inst => (runners || [])
-            .filter(run => run.instanceHash.toString() === inst.hash.toString())
-            .map(run => base58.encode(run.hash))
+        get: srv => instances
+          .filter(inst => inst.serviceHash === srv.hash)
```

Comment:
i got the following error when service:list just after creating a service (no instance and no runner started)
```
./packages/cli/bin/run service:list
 ›   Error: Cannot read property 'filter' of null
```

Reply:
The api returns:
```
➜  engine git:(dev) curl 127.0.0.1:1317/instance/list
{"height":"37","result":null}%
➜  engine git:(dev) curl 127.0.0.1:1317/runner/list
{"height":"39","result":null}%
```
it should return `"result": []`...

Reply:
doing service:list with 1 service and 1 instance but no runner is working:
```
➜  js-sdk git:(feature/lcd-server) ./packages/cli/bin/run service:list
HASH                                         SID          INSTANCES                                    RUNNERS
2msvJE8est3tTfos2wEnSQMW9oRVjUtTGDoSeyZ4x53G test-service EGyLyEVdxS5oseAUwRtrZteH9RhG1dzmR5zqimXrzdtX
```
```
➜  engine git:(dev) curl 127.0.0.1:1317/runner/list
{"height":"76","result":null}%
```

Reply:
done, the API returns always an array even if it's empty.
It was done on some resources but not all